modules/oe_content_publication/config/install/field.storage.node.oe_document.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
```

Comment:
This will maybe have a cardinality unlimited in which case we'll need to call it `oe_documents`. Let's wait until it's clarified.